content/javascript/concepts/objects/objects.md

```diff
@@ -167,7 +167,7 @@ Person {
 } 
 */
 
-person.greeting();
+console.log(person.greeting());
 // Output: "Hi, I am Elizabeth Harmon"
```

Comment:
```suggestion
// Output: "Hi, I am Elizabeth Harmon"
```

output should look like this but // Output: Hi, I am Elizabeth Harmon.

Reply:
I made changes as suggested by removing the double quotes and adding a period at the end.